extensions/wikia/UserLogin/FacebookButtonController.class.php

```diff
@@ -15,6 +15,8 @@ public function index() {
 
 		if ( $this->app->checkSkin( 'wikiamobile' ) ) {
 			JSMessages::registerPackage( 'fblogin' , array( 'wikiamobile-facebook-connect-fail' ) );
+			JSMessages::enqueuePackage( 'fblogin', 'inline' );
```

Comment:
Hmm, this still isn't working for me. I think it's because this second argument is expected to be an integer, not a string. There's a class constant we should probably be using anyways. If I update this to the following, then it seems work:

```php
JSMessages::enqueuePackage( 'fblogin', JSMessages::INLINE );
```